src/patternfly/components/Menu/menu.scss

```diff
@@ -89,6 +111,62 @@
     }
   }
 
+  &.pf-m-drilldown {
+    width: var(--pf-c-menu--m-drilldown--Width);
+    height: var(--pf-c-menu--m-drilldown--Height);
+    overflow: hidden;
+    transition: transform var(--pf-c-menu--m-drilldown--Transition--transform), height var(--pf-c-menu--m-drilldown--Transition--height);
+
+    &.pf-m-drilled-in {
+      // stylelint-disable selector-max-class
+      // target first list in menu
+      > .pf-c-menu__content > .pf-c-menu__list,
+      > .pf-c-menu__list {
+        transform: translateX(-100%);
+      }
+      // stylelint-enable
+    }
+
+    // Nested menu
+    .pf-c-menu {
+      --pf-c-menu--BoxShadow: none;
+
+      position: absolute;
+      top: calc(var(--pf-c-menu--PaddingTop) * -1);
+      left: 100%;
+      width: 100%;
+      transition: transform var(--pf-c-menu--m-drilldown__menu--Transition--transform), visibility var(--pf-c-menu--m-drilldown__menu--Transition--visibility);
+
+      &.pf-m-drilled-in {
+        transform: translateX(-100%);
+      }
+    }
+
+    .pf-c-menu__list {
+      position: relative;
+      transition: transform var(--pf-c-menu--m-drilled-in__list--Transition--transform);
+    }
+
+    .pf-c-menu__list-item {
+      position: static;
```

Comment:
Looks like this is only `position: relative;` to begin with to support the flyout variation. At least that's the only other place we define `position` (it's `postion: absolute;` on nested `.pf-c-menu` elements, which should be relative to `.pf-c-menu__list-item`. I bet we could remove this, and add `.pf-c-menu__list-item { postion: relative; }` in the flyout block here - https://github.com/patternfly/patternfly/blob/2f4ad756a10f0bbd1aa9b0f712e018621c166e51/src/patternfly/components/Menu/menu.scss#L84-L90

Reply:
Nice!